src/pages/blog/[slug].jsx

```diff
@@ -30,7 +30,7 @@ export async function getStaticProps({ params: { slug } }) {
 export default function Artikel({ frontmatter, content }) {
   return (
     <div className="w-full py-10">
-      <div className="container mx-auto prose">
+      <div className="container mx-auto prose-full text-white">
```

Comment:
```suggestion
      <article className="container mx-auto prose-full text-white">
```

changes to `article` tag instead dull non-semantic components